src/mon/MonCommands.h

```diff
@@ -869,6 +869,31 @@ COMMAND("osd pool get-quota " \
         "name=pool,type=CephPoolname ",
         "obtain object or byte limits for pool",
         "osd", "r", "cli,rest")
+COMMAND("osd pool application enable " \
```

Comment:
FYI -- CephFS integration is here [1] and rgw/rbd is forthcoming (I added them as todos above) since I needed a librados API.

[1] https://github.com/ceph/ceph/pull/15763/commits/bba43fc3c03f82efbf26d0fb07eaa0a730d08160